packages/next/next-server/lib/head.tsx

```diff
@@ -12,10 +12,7 @@ export function defaultHead(inAmpMode = false) {
   const head = [<meta charSet="utf-8" />]
   if (!inAmpMode) {
     head.push(
-      <meta
-        name="viewport"
-        content="width=device-width,minimum-scale=1,initial-scale=1"
-      />
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
I don't believe Next officially supports Safari 8, so `initial-scale` can be safely dropped.

```suggestion
      <meta name="viewport" content="width=device-width" />
```

Reply:
Sounds good, since the [default browser list](https://browserl.ist/?q=defaults) used by Next.js support Safari for iOS 12+